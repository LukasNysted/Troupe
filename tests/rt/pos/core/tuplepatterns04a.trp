let fun f () = ((),(1),(2,3)) in
  case f () of
    ((),(1),(x,3)) => x
  end