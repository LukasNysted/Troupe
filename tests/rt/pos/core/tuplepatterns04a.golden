2023-07-12T13:43:48.305Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: 2@{}%{}
